src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -38,6 +38,33 @@ public class DriveSubsystem extends SubsystemBase {
   // ENCODER DECLARATION
   private final RelativeEncoder m_leftEncoder = m_leftLeaderMotor.getEncoder();
   private final RelativeEncoder m_rightEncoder = m_rightLeaderMotor.getEncoder();
+  
+
+  private void setPidGains(){
+    leftSpeedPID.setP(0.0); //Integral, derivative, and proportional gains for the PID controller
+    leftSpeedPID.setD(0.0);
+    leftSpeedPID.setI(0.0);
+  
+    rightSpeedPID.setP(0.0);
+    rightSpeedPID.setD(0.0);
+    rightSpeedPID.setI(0.0);
+  }
+
+  double DriveDistance=0.0; //Allows setDistanceToDrive to accept an argument
+  public void setDistanceToDrive(double DriveDistance){
+
+    this.DriveDistance=DriveDistance;
+    rightSpeedPID.setReference(DriveDistance,com.revrobotics.CANSparkBase.ControlType.kPosition);
+    leftSpeedPID.setReference(DriveDistance,com.revrobotics.CANSparkBase.ControlType.kPosition);  
+
+  }
+  
+  public void setPoint(){
```

Comment:
Let's rename this `setPoint()` method to something more clear, like `setVelocity()`, or `setVelocityPID()`.

Reply:
Renamed setpoint to setvlocity.